src/Compilers/Core/Portable/Diagnostic/DiagnosticSeverity.cs

```diff
@@ -27,7 +27,12 @@ public enum DiagnosticSeverity
         /// <summary>
         /// Something not allowed by the rules of the language or other authority.
         /// </summary>
-        Error = 3
+        Error = 3,
+
+        /// <summary>
+        /// Something that can be potentially improved.
+        /// </summary>
+        Suggestion = 4,
```

Comment:
Info just represents set of messages that show up in the error list under "Messages" tab, have no visual representation (squiggles) in the editor, do not break your build by default (unless you specify `/warnaserror:InfoId` or bump it to error via a ruleset entry) and are not shown in the msbuild output (unless you bump up the verbosity to detailed or diagnostic).

All the above requirements seem to be identical for Suggestion, except for the visual representation in the editor.
I don't think anyone would really complain, if we repurpose Info to mean Suggestions if they had a special custom tag attached to it. This way even (2) is not an issue. 

My concern is we already have couple of severities that command line compile really doesn't care about: Info and Hidden, and probably even they could have been combined into a single severity with a different custom tag to indicate editor doesn't want to show these. Now we are adding another one just for the enhanced IDE experience. IMO we should be adding new entries to DiagnosticSeverity enum only if they provide additional value for all diagnostic hosts, and this doesn't seem to be the case here for command line compilation. I thought the sole purpose of adding custom tags to diagnostics was to allow adding such host specific custom information to the diagnostics.